shared/templates/rsyslog_logfiles_attributes_modify/oval.template

```diff
@@ -1,67 +1,67 @@
 <def-group oval_version="5.11">
-  <definition class="compliance" id="rsyslog_files_groupownership" version="1">
+  <definition class="compliance" id="rsyslog_files_{{{_RULE_ID }}}_ownership" version="1">
     {{{ oval_metadata("All syslog log files should be owned by the appropriate group.") }}}
 
     <criteria operator="AND">
       {{% if product in ["debian10", "debian11", "ubuntu1604"] %}}
       <extend_definition comment="rsyslog daemon is used as local logging daemon" definition_ref="package_rsyslog_installed" />
       {{% endif %}}
-      <criterion comment="Check if all system log files are owned by the appropriate group" test_ref="test_rsyslog_files_groupownership" />
+      <criterion comment="Check if all system log files are owned by the appropriate {{{ ATTRIBUTE }}}" test_ref="test_rsyslog_files_{{{ _RULE_ID }}}_ownership" />
```

Comment:
I would suggest to use `test_ref="test_{{{ _RULE_ID }}}"` and on line 98. It would make the template more generic.

Reply:
Thanks :bow: done in [ba94e4c](https://github.com/ComplianceAsCode/content/pull/9789/commits/ba94e4ceee13613c8fc2527e66d9ea265ddc5690)